packages/dev-server-rollup/test/node/fixtures/commonjs/commonjs-browser-test.js

```diff
@@ -33,7 +34,7 @@ describe('commonjs', () => {
   });
 
   it('can handle require default', () => {
-    expect(requiredDefault).to.equal('foo');
+    expect(requiredDefault.default).to.equal('foo');
```

Comment:
@43081j this feels like an error to have converted, in that this same wrapping of default is causing me some trouble with the canaries...thoughts?